internal/driver/daemon.go

```diff
@@ -323,6 +325,7 @@ func (r *RegistryDefault) ReadRouter(ctx context.Context) http.Handler {
 	for _, f := range r.defaultHttpMiddlewares {
 		n.UseFunc(f)
 	}
+	n.UseFunc(semconv.Middleware)
```

Comment:
Can we just add this to the `defaultHttpMiddlewares`?

Reply:
I already looked into it and had a some difficulties finding an appropriate place to do this. Do you have any recommendation where to best do it?

Reply:
I'll leave this as is to get this thing merged.